### PR TITLE
Add me.ppvan.psequel

### PR DESCRIPTION
**Pull request by ppvan**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: me.ppvan.psequel.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/vala/bin",
+        "prepend-ld-library-path": "/usr/lib/sdk/vala/lib",
+        "env": {}
```
Comment (hfiguiere) [inline me.ppvan.psequel.json:20]:
```suggestion
        "prepend-ld-library-path": "/usr/lib/sdk/vala/lib"
```

File: me.ppvan.psequel.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                {
+                    "type": "git",
+                    "url": "https://github.com/ppvan/psequel",
+                    "tag": "v0.1.1",
+                    "commit": "2a8f412d22b87dd706c6e31d0fa248fdd94a5c94"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "blueprint-compiler",
```
Comment (hfiguiere) [inline me.ppvan.psequel.json:48]: there should be a cleanup up to remove it all as this s only a build dependency.

File: me.ppvan.psequel.json [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                            "type": "git",
+                            "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                            "tag": "v0.10.0"
+                        }
+                    ]
+                },
+                {
+                    "name": "postgresql-libs",
+                    "build-options": {
+                        "config-opts": [
```
Comment (hfiguiere) [inline me.ppvan.psequel.json:61]: config-opts doesn't beling in `build-options`. `flatpak-builder` probably told you when you tried the build locally.

File: me.ppvan.psequel.json (hunk 2 above)
Comment (ppvan) [reply me.ppvan.psequel.json:47]: Can you give an example?
I looked other project using blueprints and they don't do anything special

Comment (hfiguiere) [reply me.ppvan.psequel.json:47]:
```json
cleanup: [
    "*"
],
```

if they don't do that they do it wrong.

Comment (ppvan) [reply me.ppvan.psequel.json:47]: thanks, i 'll commit the changes with the postgres build-options. No idea what it is yet

File: me.ppvan.psequel.json (hunk 3 above)
Comment (ppvan) [reply me.ppvan.psequel.json:60]: I think i just get rid of them then, they are no by default anyway

Comment (ppvan) [reply me.ppvan.psequel.json]: I think i just do something wrong with commit history...

File: .github/pull_request_template.md
Comment (hfiguiere) [inline .github/pull_request_template.md:1]: why was `.github` added here?

Also `CONTRIBUTING.md`, `COPYING` and `README.md`

Comment (ppvan) [reply .github/pull_request_template.md:1]: i accidently merge the master...
Can i close this and add another?

**Comment by hfiguiere** [comment]
you should force push instead.

closing PR waste reviewers time.

**Comment by ppvan** [comment]
sorry, but i thought that 'll save time, just in case i screw up again.
(I never force push before)

**Pull request closed without merging**